### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

**Comment by TheoChevalier** [comment]
There's one thing I couldn't find in the doc, it's how to hide/remove the output generated by the class.

File: src/pchevrel/Verif.php [hunk 1]
```diff
@@ -11,6 +11,7 @@ class Verif
     public $errors = [];
     public $uri;
     public $report_title;
+    public $report;
```
Comment (pascalchevrel) [inline src/pchevrel/Verif.php]: I'd prefer not having a property called $report and a public method also called report, that could lead to mistake between calls to $this->report and $this->report()

**Comment by pascalchevrel** [comment]
Put the output in a buffer that you discard:

``` php
        ob_start();
        $obj->report();
        include TEST_FILES . '/reports/' . $b;
        $this
            ->string($obj->report)
                ->isEqualTo($report);
        ob_end_clean();
```

**Comment by TheoChevalier** [comment]
Thanks for the tip :) Updated

**Comment by pascalchevrel** [comment]
Merged from my mobile while waiting for my pizza to be served :)

**Pull request merged**